projects/js-packages/idc/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ### This is a list detailing changes for the Jetpack RNA IDC package releases.
 
+## 0.10.15 - 2022-06-21
+
```

Comment:
```suggestion
- Removed testing infrastructure not used.

```

Reply:
We do sometimes, this happens when the `Comment:` field is used instead of a "public" changelog entry. I think it's fine to have at least something show up under the version though.